package.json

```diff
@@ -62,6 +62,7 @@
     ]
   },
   "dependencies": {
+    "@types/request": "^2.48.1",
```

Comment:
This should be in `devDependencies`, not `dependencies`

Reply:
Gotcha